core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -670,12 +675,16 @@ class GroupMetadataManager(brokerId: Int,
                   case unknownKey =>
                     throw new IllegalStateException(s"Unexpected message key $unknownKey while loading offsets and group metadata")
                 }
+                messagesRead += 1
+                bytesRead += record.sizeInBytes()
               }
             }
+            batchesRead += 1
             currOffset = batch.nextOffset
           }
         }
 
+        info(s"Number of messages/bytes/batches read: $messagesRead/$bytesRead/$batchesRead for topic $topicPartition")
```

Comment:
how about replacing "for topic $topicPartition" by "for partition $topicPartition"

Reply:
done..